### PR TITLE
[EAGLE-3760] - example for a torchserve detector model

### PR DESCRIPTION
**Pull request by phatvo9**

### CODE REVIEW

**Comment by ackizilkale** [comment]
We can not use this model. Can we use yolof going forward?

**Comment by phatvo9** [comment]
> We can not use this model. Can we use yolof going forward?

Change to `faster-rcnn` as it's one of the torchserve examples

File: clarifai/models/model_serving/examples/visual_detection/faster-rcnn_torchserve/1/inference.py
Comment (HarmitMinhas96) [inline clarifai/models/model_serving/examples/visual_detection/faster-rcnn_torchserve/1/inference.py:1]: Could you write a brief README for the faster-rcnn_torchserve subfolder so someone could know all the steps they'd need to take to go from this example to their own torchserve model uploaded to the clarifai api?
E.g:
1. Download torchserve model
2. Unzip .mar and put it into folder x
3. Update inference.py if your model outputs are different
4. Run zip
5. Upload to your own cloud bucekt
6. Call upload with cloud url

File: clarifai/models/model_serving/examples/visual_detection/Readme.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+unzip faster_rcnn.mar -d ./faster-rcnn_torchserve/1/model_store/hub/checkpoints
+```
+
+```bash
+# in model_store you will have
+model_store/
+├── MAR-INF
+│   └── MANIFEST.json
+├── model.py
+└── fasterrcnn_resnet50_fpn_coco-258fb6c6.pth
```
Comment (HarmitMinhas96) [inline clarifai/models/model_serving/examples/visual_detection/Readme.md:47]: Is this .pth supposed to be in `/model_store/hub/checkpoints` as per the unzip command above?

Comment (phatvo9) [reply clarifai/models/model_serving/examples/visual_detection/Readme.md:47]: Thanks for the catch, I've updated

**Review by HarmitMinhas96 — APPROVED** [review]

**Pull request merged**